helper_crates/vtable/macro/macro.rs

```diff
@@ -764,7 +766,7 @@ and implements HasStaticVTable for it.
                         &$ident
                     }
                 }
-            }
+            };
```

Comment:
I'm surprised that this went unnoticed. Do you have any idea why?

Reply:
This should not be here. I was playing with the macro and added it as I was trying out an alternate pattern or something in addition to the existing one.